### PR TITLE
Update Fabric8 version and usage.

### PR DESCRIPTION
**Pull request by corneil**
Update Fabric8 version and usage.
Added version for json-path 2.9.0

Unit Tests and ITs pass locally

### CODE REVIEW

File: spring-cloud-deployer-kubernetes/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesScheduler.java [hunk 1]
```diff
@@ -158,10 +160,10 @@ public void validateScheduleName(ScheduleRequest request) {
 
 	@Override
 	public void unschedule(String scheduleName) {
-		boolean unscheduled = this.client.batch().v1().cronjobs().withName(scheduleName).delete();
+		List<StatusDetails> unscheduled = this.client.batch().v1().cronjobs().withName(scheduleName).delete();
 
-		if (!unscheduled) {
-			throw new SchedulerException("Failed to unschedule schedule " + scheduleName + " does not exist.");
+		if (unscheduled.isEmpty()) {
```
Comment (onobc) [inline spring-cloud-deployer-kubernetes/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesScheduler.java:165]: Is `unscheduled` always non-null?

File: .github/workflows/ci-pr.yml [hunk 2]
```diff
@@ -27,7 +27,7 @@ jobs:
     # build
     - name: Build
       run: |
-        mvn -U -B clean package -T 0.5C
```
[the remark this answers is not in the document]
Comment (corneil) [reply .github/workflows/ci-pr.yml:30]: -T 0.5C in Github means it is only using one core so it has no real effect.

File: pom.xml [hunk 3]
```diff
@@ -57,10 +57,11 @@
 		<maven-resolver.version>1.9.18</maven-resolver.version>
 		<maven-wagon.version>3.5.3</maven-wagon.version>
 		<commons-io.version>2.15.1</commons-io.version>
-		<kubernetes-fabric8-client.version>5.12.4</kubernetes-fabric8-client.version>
+		<kubernetes-fabric8-client.version>6.9.2</kubernetes-fabric8-client.version>
 		<spring-boot.version>3.2.2</spring-boot.version>
 		<maven.compiler.source>${java.version}</maven.compiler.source>
 		<maven.compiler.target>${java.version}</maven.compiler.target>
+		<json-path.version>2.9.0</json-path.version>
```
[the remark this answers is not in the document]
Comment (corneil) [reply pom.xml:64]: We will remove when SB is on 2.9.0

File: spring-cloud-deployer-kubernetes/pom.xml [hunk 4]
```diff
@@ -85,13 +82,13 @@
 		<dependency>
 			<groupId>com.squareup.okhttp3</groupId>
 			<artifactId>okhttp</artifactId>
-			<version>${okhttp.version}</version>
+<!--			<version>${okhttp.version}</version>-->
```
[the remark this answers is not in the document]
Comment (corneil) [reply spring-cloud-deployer-kubernetes/pom.xml:85]: Sure

**Pull request merged**